### PR TITLE
added actions in bookings controller

### PR DESCRIPTION
**Pull request by chihoko12**
i migrated the booking tables. The controllers actions i'll check again.

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,8 +1,49 @@
...
+    @bookings = Booking.all
+  end
 
   def new
+    @booking = Booking.new
   end
 
   def create
+    @booking = Booking.new(booking_params)
+    @booking.user = @user
```
Comment (sorina-duca) [inline app/controllers/bookings_controller.rb:14]: You need to find the user. According to the routes, the user should be in params[:user_id]

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -1,8 +1,49 @@
...
+  def index
+    @bookings = Booking.all
+  end
 
   def new
+    @booking = Booking.new
   end
 
   def create
+    @booking = Booking.new(booking_params)
```
Comment (sorina-duca) [inline app/controllers/bookings_controller.rb:13]: @chihoko12 , how do you get the user and the boat in order to pass them to the Booking? I think according to the routes, the user is params[user_id]. We can't really check in the browser for now, I guess.

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -1,8 +1,49 @@
...
+
+  def destroy
+    @booking.destroy
+    redirect_to user_booking_path(@booking.user)
+  end
+
+  private
+
+  def find_booking
+    @booking = Booking.find(params[:booking_id])
```
Comment (sorina-duca) [inline app/controllers/bookings_controller.rb:42]: I believe he booking is params[:id]. Please check the routes

Comment (chihoko12) [reply app/controllers/bookings_controller.rb]: True, thank you.

File: app/controllers/bookings_controller.rb (hunk 2 above)
Comment (chihoko12) [reply app/controllers/bookings_controller.rb:15]: i add authenticate_user for new and create methods and set @booing.user = take current_user.id.

**Pull request merged**